lib/github_service.rb

```diff
@@ -12,9 +12,16 @@ module GithubService
   #
   class << self
     def add_comments(fq_repo_name, issue_number, comments)
-      Array(comments).each do |comment|
+      # add_status requires first comment due to its URL
+      Array(comments).map do |comment|
         add_comment(fq_repo_name, issue_number, comment)
-      end
+      end.first
```

Comment:
I like the idea of returning the comments, but from a clean standpoint, this method should just return all of the comments.  If the caller only needs the first one, they should be the one to call .first.  This way you also don't need the comment describing why you need the .first.